show/src/main/resources/application.yml

```
@@ -0,0 +1,6 @@
+spring:
+  application:
+    name: show-service
+
+  config:
+    import: configserver:${CONFIG_SERVER_URL:http://localhost:19000}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)